  
<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>
  <arg name="custom_config" default="" />
  <group ns="$(arg uav_name)">
  <node name="fusion_radiation" pkg="nodelet" type="nodelet"  args="standalone fusion_radiation/fusion_radiation" output="screen">

    <rosparam file="$(find fusion_radiation)/config/default.yaml" command="load" />
    <rosparam if="$(eval not arg('custom_config') == '')" file="$(arg custom_config)" />

    <param name="uav_name" value="$(arg uav_name)" />
    <remap from="~markers_visualizer" to="markers_visualizer2"/>
    
  </node>
  </group>

</launch>
